extensions/coffee/src/index.tsx

```diff
@@ -1,44 +1,35 @@
-import { environment, MenuBarExtra, updateCommandMetadata } from "@raycast/api";
+import { LaunchProps, MenuBarExtra } from "@raycast/api";
 import { useExec } from "@raycast/utils";
-import { useEffect } from "react";
 import { stopCaffeinate, startCaffeinate } from "./utils";
 
-export default function Command() {
-  const hasLaunchContext = environment.launchContext?.caffinated !== undefined;
+export default function Command(props: LaunchProps) {
+  const hasLaunchContext = props.launchContext?.caffeinated !== undefined;
 
-  const { isLoading, data, mutate } = useExec("ps aux | pgrep caffeinate", [], {
+  const { isLoading, data, mutate } = useExec("pgrep caffeinate", [], {
```

Comment:
I think the original intention was for it to be `ps aux | grep caffeinate`, but `pgrep caffeinate` does the same thing.